packages/vite/src/node/plugins/css.ts

```diff
@@ -1866,37 +1945,24 @@ export interface StylePreprocessorResults {
   deps: string[]
 }
 
-const loadedPreprocessors: Partial<
-  Record<PreprocessLang | PostCssDialectLang, any>
+const loadedPreprocessorPath: Partial<
+  Record<PreprocessLang | PostCssDialectLang, string>
 > = {}
 
-// TODO: use dynamic import
-const _require = createRequire(import.meta.url)
-
-function loadPreprocessor(lang: PreprocessLang.scss, root: string): typeof Sass
-function loadPreprocessor(lang: PreprocessLang.sass, root: string): typeof Sass
-function loadPreprocessor(lang: PreprocessLang.less, root: string): typeof Less
-function loadPreprocessor(
-  lang: PreprocessLang.stylus,
-  root: string,
-): typeof Stylus
-function loadPreprocessor(
-  lang: PostCssDialectLang.sss,
-  root: string,
-): PostCSS.Parser
-function loadPreprocessor(
+function loadPreprocessorPath(
   lang: PreprocessLang | PostCssDialectLang,
   root: string,
-): any {
-  if (lang in loadedPreprocessors) {
-    return loadedPreprocessors[lang]
+): string {
+  const cached = loadedPreprocessorPath[lang]
+  if (cached) {
+    return cached
   }
   try {
     const resolved = requireResolveFromRootWithFallback(root, lang)
-    return (loadedPreprocessors[lang] = _require(resolved))
+    return (loadedPreprocessorPath[lang] = resolved)
   } catch (e) {
+    const installCommand = getPackageManagerCommand('install')
     if (e.code === 'MODULE_NOT_FOUND') {
```

Comment:
Could the `installCommand` be moved withing this if condition like before? Looks like it's not used anywhere else.